src/app/shared/orgschema.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { Organization } from './openapi';
+
+export interface OrganizationSchema {
+  '@context': string;
+  '@type': string;
+  contactPoint?: Array<ContactPoint>;
+  description?: string;
+  location?: string;
+  logo?: string;
+  name: string;
+  url: string;
+}
+
+export interface ContactPoint {
+  // Represents the contact email displayed on the organization page
+  '@type': string;
+  email: string;
+}
+
+@Injectable()
+export class OrgschemaService {
```

Comment:
rename to OrgSchemaService and change filename to org-schema.service.ts for consistency.